lib/plugins/wallet/monerod/monerod.js

```diff
@@ -175,12 +156,20 @@ function newFunding (account, cryptoCode, settings, operatorId) {
         balanceRes,
         fetch('create_address', { account_index: 0 })
       ]))
-    .then(([balanceRes, addressRes]) => ({
-      fundingPendingBalance: BN(balanceRes.balance).minus(balanceRes.unlocked_balance),
-      fundingConfirmedBalance: BN(balanceRes.unlocked_balance),
-      fundingAddress: addressRes.address
-    }))
-    .catch(err => handleError(err))
+    .then(([balanceRes, addressRes]) => Promise.all([
+        balanceRes,
+        addressRes,
+        fetch('get_transfers', { pool: true, account_index: 0 })
+      ]))
+    .then(([balanceRes, addressRes, transferRes]) => {
```

Comment:
Any particular reason to split the 3 calls in 3 different `.then()`s? They look independent of each other to me. Feel free to ignore if you don't think it's worth the effort.